### PR TITLE
add exception handling programs

### PR DESCRIPTION
**Pull request by aayush105**

### CODE REVIEW

File: programs/ExceptionHandling/q1.cpp
Comment (thegauravgiri) [inline programs/ExceptionHandling/q1.cpp:1]: Use a simple case like when negative throw error and catch the negative number then display it.

File: programs/ExceptionHandling/q10.cpp
Comment (thegauravgiri) [inline programs/ExceptionHandling/q10.cpp:1]: Need another example with more simple and complete code.

File: programs/ExceptionHandling/q4.cpp
Comment (thegauravgiri) [inline programs/ExceptionHandling/q4.cpp:1]: Take two numbers and divide them, if the denominator is 0 throw or if any number is negative throw exception.

File: programs/ExceptionHandling/q7.cpp [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// WAP to show try catch throw in user define classes type.
+
+#include <iostream>
+using namespace std;
+
+class MyException
+{
```
Comment (thegauravgiri) [inline programs/ExceptionHandling/q7.cpp:8]: add a constructor so that, it can handle the exception from inside.

**Review by thegauravgiri — CHANGES_REQUESTED** [review]

**Review by nawarajshah — APPROVED** [review]

**Review by thegauravgiri — APPROVED** [review]
LGTM!

**Pull request merged**